### PR TITLE
CFY-3040 passing old agent version to install_agent script

### PR DESCRIPTION
**Pull request by konradxyz**

### CODE REVIEW

File: cloudify_agent/operations.py [hunk 1]
```diff
@@ -262,10 +268,12 @@ def create_agent_from_old_agent(install_agent_timeout=300):
     os.environ['CELERY_BROKER_URL'] = broker_url
     try:
         celery_client = celery.Celery(broker=broker_url, backend=broker_url)
-        app_conf = {
-            'CELERY_TASK_RESULT_EXPIRES': defaults.CELERY_TASK_RESULT_EXPIRES
-        }
-        celery_client.conf.update(**app_conf)
+        if new_agent['old_agent_version'] != '3.2':
+            app_conf = {
```
Comment (simon-bar) [inline cloudify_agent/operations.py]: why not
celery_client.conf.update(CELERY_TASK_RESULT_EXPIRES=defaults.CELERY_TASK_RESULT_EXPIRES)
?

File: cloudify_agent/operations.py [hunk 2]
```diff
@@ -238,7 +240,7 @@ def create_new_agent_dict(old_agent, suffix=None):
     fields_to_delete = ['name', 'queue', 'workdir', 'agent_dir', 'envdir',
```
Comment (simon-bar) [inline cloudify_agent/operations.py]: Maybe

if not suffix:
    suffix = str(uuid.uuid4())

?

File: cloudify_agent/operations.py [hunk 3]
```diff
@@ -246,6 +248,10 @@ def create_new_agent_dict(old_agent, suffix=None):
     configuration.reinstallation_attributes(new_agent)
     new_agent['broker_url'] = _get_broker_url(new_agent)
     new_agent['manager_file_server_url'] = get_manager_file_server_url()
+    new_agent['version'] = _VERSION
```
Comment (simon-bar) [inline cloudify_agent/operations.py]: Is this necessary here (version)?

**Comment by konradxyz** [comment]
Changes introduced in https://github.com/codilime/cloudify-agent/commit/e254b83df05f5cef227757b74901774446e80565

**Pull request closed without merging**